resources/js/common/components/UserAvatar/UserAvatar.test.tsx

```diff
@@ -0,0 +1,73 @@
+import { faker } from '@faker-js/faker';
+
+import { render, screen } from '@/test';
+
+import { UserAvatar } from './UserAvatar';
+
+describe('Component: UserAvatar', () => {
```

Comment:
How are these tests run? I don't see any changes to add it to CI or pre-push

Reply:
It all works seamlessly with our existing Vitest setup. Try `npm run test`.

Reply:
I just noticed that Sail was not using the correct version of Node.js. In https://github.com/RetroAchievements/RAWeb/pull/2502/commits/399dbd6aefc731b377d7a2b1fe76bae574145353 I've updated the `laravel.test` container to use Node 20.

Now you should be able to do:
```bash
sail npm install
sail npm run test
```